src/test/resources/features/AgentUserJourneys.feature

```diff
@@ -53,3 +53,74 @@ Feature: Agent user journeys
     When I select back link
     When I click Report Pillar 2 top-up taxes link
     Then I should navigate to ASA Home Page
+
+
+  Scenario: 2 - Agent user accessing dashboard features after login
+    Given Agent User logs in with existing entity group HMRC-AS-AGENT, AgentReference and 1234 for Pillar2 service
+    And I add delegated enrolment with HMRC-PILLAR2-ORG, PLRID, XMPLR0012345674 and pillar2-auth for Pillar2 service
+    Then I should be on ASA Pillar2 Input Page
+    And I provide ASA Pillar2 ID as XMPLR0012345674
+    And I click on Continue button
+    Then I should navigate to ASA Confirmation Page
+    And I click on Continue button
+    Then I should navigate to ASA Dashboard page
+    And I click Make a payment link
+    Then I should navigate to Agent Make a Payment page
+    And I select back link
+    Then I should navigate to ASA Dashboard page
+    And I click View and amend contact details link
+    Then I should navigate to Agent Contact Detail Summary Page
+    And I should see sub section 1 as First contact
+    And I should see sub section 2 as Second contact
```

Comment:
We can remove the content validation as its already being validated in unit Tests.